### PR TITLE
fix: retrieve metrics_token secret value from github secrets

### LINKED ISSUE #142: Bug: Task Definition for Server Metrics is not passing secret value.
The secret value should not be in a container definition since the entire thing is a container definition. 

Also the folder is named improperly.

### PR DESCRIPTION
**Pull request by mohamed-cds**
Fixes #142

### CODE REVIEW

**Review by CalvinRodo — APPROVED** [review]
:taco::cat:

File: env/staging/ecs/terragrunt.hcl [hunk 1]
```diff
@@ -69,6 +69,7 @@ inputs = {
   memory                              = 1024
   aggregate_metrics_table_arn         = dependency.dynamodb.outputs.aggregate_metrics_arn
   schedule_expression                 = "rate(24 hours)"
+  metrics_token                       = "${get_env("TF_VAR_METRICS_TOKEN")}"
```
Comment (patheard) [inline env/staging/ecs/terragrunt.hcl:72]: I had thought the `TF_VAR_METRICS_TOKEN` environment variable would get automatically read in here, but I haven't tested with a terragrunt.hcl file.

**Review by patheard — COMMENTED** [review]
Comment about the env var.

File: env/staging/ecs/terragrunt.hcl (hunk 1 above)
Comment (mohamed-cds) [reply env/staging/ecs/terragrunt.hcl:72]: i'll test it out and see what happens when i remove that line and introduce a TF_VAR_metrics_token. Maybe the case is the reason why it wasn't working and forced me to do this.

**Pull request merged**